lib/google_scraping_web/templates/keyword/show.html.heex

```diff
@@ -1,3 +1,73 @@
-<h2>Keywords details <%= @keyword.name %></h2>
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item">
+      <%= link(gettext("Keywords"),
+        to: Routes.keyword_path(@conn, :index)
+      ) %>
+    </li>
+    <li class="breadcrumb-item active" aria-current="page">
+      <%= gettext("Keyword '%{name}' details", %{name: @keyword.name}) %>
+    </li>
+  </ol>
+</nav>
 
-<p>Coming soon!</p>
+<div class="card">
+  <div class="card-header text-center">
+    <h2><%= @keyword.name %></h2>
+  </div>
+  <div class="card-body">
+    <dl class="row">
+      <dt class="offset-sm-3 col-lg-5"><%= gettext("Status") %></dt>
+      <dd class="col-lg-4"><%= @keyword.status %></dd>
+      <%= if @keyword.status == :completed do %>
+        <dt class="offset-sm-3 col-lg-5">
+          <%= gettext("Number of AdWords advertisers in the top position") %>
+        </dt>
+        <dd class="col-lg-4"><%= @keyword.ad_top_count %></dd>
+        <dt class="offset-sm-3 col-lg-5">
+          <%= gettext("The total number of AdWords advertisers on the page") %>
+        </dt>
+        <dd class="col-lg-4"><%= @keyword.ad_total_count %></dd>
+        <dt class="offset-sm-3 col-lg-5">
+          <%= gettext("URLs of the AdWords advertisers in the top position") %>
```

Comment:
Should we say the `Number of AdWords advertiser's URLs in the top position`?

Reply:
Thanks. It was fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/50/commits/a40687f81547f185765054526fe0760518a29b3c